docs/user-guide/admin-cli.md

```diff
@@ -11,13 +11,61 @@ of all virtual disks created and in use on the host.  For these reasons, an admi
 created that runs on the ESXi host and that provides access to information not visible from the
 Docker CLI.
 
-The admin cli also enables ESX admins to implement tenancy.
+The admin CLI also enables ESX admins to implement access control and basic storage quotas.
+
+Admin CLI is located at `/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py`. It supports `--help`  at
+every command and sub-command. e.g.
+
+**NOTE** PXE Boot of ESX host is not supported, as the code relies on configuration / authorization databse (aka "config DB") (or symlink) being in /etc/vmware/vmdksops/auth-db. If the host is PXE  booted, access control will be turned off, as the code will interpret the lack of DB or link at the above address as an instruction to provide no access control.
```

Comment:
PXE boot can be mentioned as "stateless ESXi  is not supported ....". Can the VIB install locate the DB on any of the datastores available vs. on /etc/..... ? The DB may not be large but root fs on ESX may not be the place to locate the DB (may want to move the DB to a datastore long term).